pebblo/app/service/local_ui/loader_apps.py

```diff
@@ -287,6 +314,23 @@ def get_findings_for_loader_app(self, app_data):
         )
         return app_details.model_dump()
 
+    def _create_loader_app_model(self, app_list):
```

Comment:
Lets add some comments and param and output type

Reply:
Fixed.